### ISSUE TITLE
Hello

### ISSUE THREAD

**Issue by CatsArmy** [TisRyno/LethalProgression#7]
I see you've fixed some bugs noice I'll be real I don't really want to maintain this project. atm I'm just waiting for stoneman to come back for it.
just thought to let you know.

Btw if you published this the TS just tell me and I'll deprecate my patch of the og mod 

Btw I wouldn't use my save system without testing it cus I wasn't sure if it even worked can't even lie I only did that change because I thought it was the root issue to one of the errors in the log caused by the mod so yeah

**Comment by TisRyno** [comment]
Hey @CatsArmy

Thanks for reaching out, I only wanted to apply the patch for GUI issue which causes players to be unable to interact anymore.

However it has opened a can of worms when attempting to publish the package and I've spent all day trying to track down the root cause, it seems to relate to the skillsmenu asset bundle.

Hopefully I can get a working copy out soon and I'll be happy to publish it.

**Comment by TisRyno** [comment]
To update to this.

I spent today backtracking a bunch of changes and have successfully fixed all the issues, your original bug fix was implemented and I've fixed the GUI issue which caused levelling up to stop and causing players to be unable to interact.

I'm happy to take over the maintaining now if you'd prefer as I've released it on Thunderstore at 1.6.0

Perhaps update the README to push people to my copy?
